docs/cli/overview.md

```diff
@@ -32,6 +32,14 @@ Verify the available commands and options:
 expressots --help
 ```
 
+### Version
```

Comment:
You can remove this section, as we don't have version command anymore.
All data from the project is coming form the `info` command
expressots info

Reply:
OK, I'll do it. :)